crates/core/tedge_api/src/mqtt_topics.rs

```diff
@@ -628,6 +631,18 @@ pub enum ChannelFilter {
     CommandMetadata(OperationType),
 }
 
+pub fn command_id_requester_with_timestamp(requester: &str) -> Result<String, error::Format> {
```

Comment:
I would really avoid to return an error here => any formatting error must be handled internally by this function.
It can be using another format e.g. an unix timestamp instead.

Reply:
I am wondering, can't we just `unwrap` this value? It is delivered by `OffsetDateTime::now_utc()` so we should always be able to format it into `RFC3339`.

Reply:
`unwrap()` is indeed okay.